mittab/apps/tab/forms.py

```diff
@@ -474,3 +474,9 @@ def score_panel(result, discard_minority):
     pprint.pprint(ranked)
 
     return ranked, final_winner
+
+
+class ArchiveDownloadForm(forms.Form):
+    name = forms.CharField(max_length=100, required=True)
```

Comment:
Can we restrict this to alpha-numeric values with no spaces? Since it goes directly into an http header I'd like it to be super sanitized.